lib/ProductOpener/Import.pm

```diff
@@ -1023,69 +1024,98 @@ EMAIL
 							$new_field_value =~ s/litre|litres|liter|liters/l/i;
 							$new_field_value =~ s/kilogramme|kilogrammes|kgs/kg/i;
 					}
-
+					
 					$new_field_value =~ s/\s+$//g;
-					$new_field_value =~ s/^\s+//g;
+					$new_field_value =~ s/^\s+//g;					
+					
+					# Some fields like "obsolete" can have yes/no values
+					if ($field eq "obsolete") {
+						if ($new_field_value =~ /^\s*(1|y|yes|o|oui|s|si)\s*$/i) {
```

Comment:
Can this be factored ?

Reply:
We can move the strings in a hash table above the function as we do for other regexps for ingredients analysis etc.

Reply:
It's done.